frontend/src/features/mapSideBar/tabs/propertyDetails/update/UpdatePropertyDetailsForm.tsx

```diff
@@ -86,6 +87,7 @@ export const UpdatePropertyDetailsForm: React.FC<IUpdatePropertyDetailsFormProps
 
   return (
     <StyledForm>
+      <UnsavedChangesPrompt />
```

Comment:
This will prevent navigating out of the form if there are unsaved changes but don't we want to also confirm upon cancel button click?

Reply:
when clicking the cancel button the action attempts to change the route, which also triggers this.